AUTHORS.txt

```diff
@@ -20,6 +20,7 @@ Former maintainers:
 Contributors:
 
 - Peter Leonov <gojpeg@gmail.com>
+- Vanessa Sochat <@vsoch>
```

Comment:
Are you opposed to adding your email?  This list is used in other places where it's expected that these are emails (npm contributor list, etc)... or perhaps NPM doesn't care?  I'm not sure.

Reply:
I’d prefer to use my GitHub alias, if it’s alright with you! I try to minimize the random public places where my email might be scraped, and with the alias you can still easily find and ping me anywhere on GitHub.

Reply:
This gets split out later into NPM contributors... perhaps a URL would be better?

https://docs.npmjs.com/files/package.json#people-fields-author-contributors

Otherwise I think in NPM we'd be stuck with including nothing but your name...

Reply:
> Both email and url are optional either way.

Sounds ok to me!